cxx-qt-gen/src/gen_rs.rs

```diff
@@ -111,6 +111,82 @@ impl RustType for QtTypes {
     }
 }
 
+fn generate_invokable_cxx_declaration(obj: &QObject, i: &Invokable) -> TokenStream {
+    let rust_class_name = &obj.original_rust_struct.ident;
+    let cpp_class_name = format_ident!("{}Qt", rust_class_name);
+
+    // Cache the ident and parameters as they are used multiple times later
+    let (ident, ident_cpp_str) = (
+        &i.ident_wrapper.rust_ident,
+        i.ident_wrapper.cpp_ident.to_string(),
+    );
+    let parameters = &i.parameters;
+    let mutablility = if i.mutable {
+        Some(quote! { mut })
+    } else {
+        None
+    };
+
+    // TODO: invokables need to also become freestanding functions that
+    // take as input a reference to both the Rs class and the CppObject
```

Comment:
not sure which usecase this is? Can't you just use the rust/rust_mut methods from the C++ pointer? but for future branches.

Reply:
Ah lol i see it's an old comment that has moved, so yup fine for now as this code will be ripped out anyway.